omero/sysadmins/index.rst

```diff
@@ -84,12 +88,12 @@ performance and security.
     config
     server-syslog
 
-*****************************
-Managing OMERO users and data
-*****************************
+**************
+Managing OMERO
+**************
 
-This section contains details on how user roles and permissions work in OMERO.
-New in OMERO 5.4.0, full administrators can now create restricted
+This section contains details on how to manage users, groups and data access
+in OMERO. New in OMERO 5.4.0, full administrators can now create restricted
```

Comment:
``full administrators can now create...`` -> not precise, restricted administrators can create restricted administrators too (but I understand we need to keep it simple here)

Reply:
A full administrator needs to do it the first time though and that's too much detail for an intro sentence.

Reply:
yes, probably too much detail and complication, let's leave it as it is